ITISFoundation/osparc-simcore
# ♻️ Updated models for job/studies metadata in public-api

## What do these changes do?

- ♻️ Updates jobs and studies metadata public api  (OAS)
- ♻️ ruff cleanup
-  🔨 script to produce ERD of postgres tables database

## Related issue/s

- Implementing https://github.com/ITISFoundation/osparc-simcore/issues/4313


## How to test

- ERD runs via a `make` recipe and produces both a png and svg images
```cmd
cd packages/postgres-database
make install-dev
make auto-doc
```
and should see ![postgres-database-erd-ignore](https://github.com/ITISFoundation/osparc-simcore/assets/32402063/9970aec5-2b1b-41ae-95e5-d94580a2de36)


- api-server: test driven by `services/api-server/tests/unit/api_studies/test_api_routes_studies_jobs.py`

## DevOps 
None
